sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlArrayTest.java

```diff
@@ -80,11 +81,11 @@ public void testSelectArrayValue() {
   public void testProjectArrayField() {
     PCollection<Row> input = pCollectionOf2Elements();
 
-    RowType resultType =
+    Schema resultType =
         RowSqlType
             .builder()
             .withIntegerField("f_int")
-            .withArrayField("f_stringArr", SqlTypeCoders.VARCHAR)
+            .withArrayField("f_stringArr", SqlTypeName.VARCHAR)
```

Comment:
I think that we should not use Calcite types in the public APIs. Especially when the schema is now in core.

Reply:
This builder will be removed